client/src/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,257 @@
+import React, { useState } from 'react';
+import './tabs.scss';
+import PropTypes from 'prop-types';
+import { Button } from '../button/Button/Button';
+import { Dropdown } from '../form/Dropdown/Dropdown';
+
+const Tabs = ({ tabs }) => {
+  return (
+    <div>
+      <div className="Tabs-desktop">
+        <TabsDesktop tabs={tabs} />
+      </div>
+      <div className="Tabs-mobile">
+        <TabsMobile tabs={tabs} />
+      </div>
+    </div>
+  );
+};
+
+const TabsDesktop = ({ tabs }) => {
```

Comment:
I think keeping the same format for both web and mobile should be okay
It's generally bad practice to load/unload component from resizing
The desktop format should work okay on mobile no?